Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md

```diff
@@ -130,7 +130,7 @@ For **Umbraco v7.7.3+**
 
 	<add key="umbracoLocalTempStorage" value="EnvironmentTemp" />
 
-This will set Umbraco to store `umbraco.config` and the other Umbraco TEMP files in the environment temporary folder. More info on this setting is available [here](../../../Reference/Config/webconfig/index.md#umbracolocaltempstorage-umbraco-v773)
+This will set Umbraco to store `umbraco.config` and the other Umbraco TEMP files in the environment temporary folder. More info on this setting is available [here](../../../../Reference/Config/webconfig/index.md#umbracolocaltempstorage-umbraco-v7.7.3)
```

Comment:
The path bit is a good fix but I'm confused by the anchor target, I can't see where you got that one?

Reply:
The hash is how the anchor links on our (https://our.umbraco.org/documentation/Reference/Config/webconfig/#umbracolocaltempstorage-umbraco-v7.7.3)

I will update it to #umbracolocaltempstorage-umbraco-v773 as per the repo anchor